ethereumj-core/src/main/java/org/ethereum/net/eth/handler/Eth62.java

```diff
@@ -195,8 +194,8 @@ public synchronized void sendStatus() {
             totalDifficulty = blockchain.getTotalDifficulty();
         }
 
-        StatusMessage msg = new StatusMessage(protocolVersion, networkId,
-                ByteUtil.bigIntegerToBytes(totalDifficulty), bestHash, config.getGenesis().getHash());
+        StatusMessage msg = new StatusMessage(protocolVersion, networkId,  // Casper genesis from config is not final
```

Comment:
Maybe this change would gone if contracts' code was a part of genesis file

Reply:
Yeah, it should gone.